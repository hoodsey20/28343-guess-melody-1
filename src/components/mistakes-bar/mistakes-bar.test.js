import React from 'react';
import renderer from 'react-test-renderer';

import MistakesBar from './mistakes-bar';

it(`MistakesBar is rendered correctly`, () => {
  const tree = renderer.create(<MistakesBar
    mistakes={3}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
