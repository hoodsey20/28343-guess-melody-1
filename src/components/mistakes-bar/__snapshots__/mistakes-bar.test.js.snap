// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MistakesBar is rendered correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
